participant/docker-compose.yml

```
@@ -54,6 +56,7 @@ services:
       S3_ENDPOINT_URL: http://s3-local:9000
       AWS_ACCESS_KEY_ID: "e2e_minio"
       AWS_SECRET_ACCESS_KEY: "incredible_local_secret_phrase"
+      MATOMO_URL_BASE: "https://analytics.wic-services.org/"
```

Let's not do this. This will send local development events to the actual prod service, so that will pollute our data.

```suggestion
      MATOMO_URL_BASE: "http://localhost:8080/"
```